source/validate.cpp

```diff
@@ -287,6 +292,10 @@ spv_result_t spvValidateIDs(const spv_instruction_t* pInsts,
 
 namespace {
 
+// Shame
+#define CHECK_RESULT(EXPRESSION) \
```

Comment:
This is now no different than `spvCheckReturn`.  Also, it contradicts the style established in other code, where we don't use a macro at all, opting to inline the `if` instead.